apps/web/src/server/applications/pages/create-new-case/case/__tests__/applications-create-case.test.js

```diff
@@ -217,17 +222,6 @@ describe('applications create', () => {
 						expect(element.innerHTML).not.toContain('checked');
 					});
 				});
-				describe('Not provided', () => {
```

Comment:
don't you need to test this anymore?

Reply:
no, this was created before the draft/non-draft guard was implemented. Now, there is a default beahviour (an error, of course) that is triggered when the ID of the case doesn't match with any draft case.
This test was making sure that, if you don't provide an ID to the "create-new-case/sector" page, you get an error, which is correct, but now the error you get is different and this test doesnt really make much sense anymore.